### PR TITLE
Ascii art

### PR DESCRIPTION
**Pull request by jbernardo95**
In response to #7.

### CODE REVIEW

**Comment by jbernardo95** [comment]
@tiagonbotelho Review please.

File: lib/need_for_type/display_window.rb [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
   class DisplayWindow < Window
+    @@need_for_type_ascii_art =
+" _   _               _    __             _______               
+| \\ | |             | |  / _|           |__   __|              
+|  \\| | ___  ___  __| | | |_ ___  _ __     | |_   _ _ __   ___ 
+| . ` |/ _ \\/ _ \\/ _` | |  _/ _ \\| '__|    | | | | | '_ \\ / _ \\
+| |\\  |  __/  __/ (_| | | || (_) | |       | | |_| | |_) |  __/
+|_| \\_|\\___|\\___|\\__,_| |_| \\___/|_|       |_|\\__, | .__/ \\___|
+                                               __/ | |         
+                                              |___/|_|"
```
Comment (tiagonbotelho) [inline lib/need_for_type/display_window.rb]: http://patorjk.com/software/taag/#p=display&f=Slant&t=Need%20for%20Type perhaps this one is better 😄

**Comment by jbernardo95** [comment]
@tiagonbotelho Updated ;)

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
```
Comment (tiagonbotelho) [inline CHANGELOG.md:5]: We should change this to log each version not added improved and fixed. This should be bumped for each version we have can you change that @jbernardo95 ?

**Review by tiagonbotelho — CHANGES_REQUESTED** [review]
Just left a final comment and it is good to go! Good job @jbernardo95 👍

File: CHANGELOG.md (hunk 2 above)
Comment (jbernardo95) [reply CHANGELOG.md:5]: @tiagonbotelho I think I didn't quite get what you said, can you elaborate ?

Comment (tiagonbotelho) [reply CHANGELOG.md:5]: I will merge this and make it myself :) no worries

**Pull request merged**